app/views/shared/_header.html.haml

```diff
@@ -1,24 +1,24 @@
 %header#content-header
   %h1#logo
-    %a{ href: '/', target: '_self' }
+    %a{ href: '/', target: '_self', aria: { label: t('branding.logo') } }
       %img{ src: asset_path(SETTINGS[:site_logo]), alt: "#{SETTINGS[:site_title]} Logo" }
   %h2= t('branding.header', location: t('branding.location'))
   %nav
     %ul
       %li
-        %a.popup-trigger.popup-default#about-btn{ href: '/about#about-box' }
+        %a.popup-trigger.popup-default#about-btn{ href: '/about#about-box', aria: { label: t('navigation.about') } }
```

Comment:
Probably we are missing the translation for this one: 
![image](https://github.com/user-attachments/assets/7b55589b-c690-4f91-83d5-8768b09c3711)

Reply:
Actually the 3 of them are missing translations: 
![image](https://github.com/user-attachments/assets/4dafb2c8-5c43-4ab6-a305-12fd9e01f730)